tap_linkedin_ads/client.py

```diff
@@ -51,6 +51,10 @@ class LinkedInInternalServiceError(LinkedInError):
     pass
 
 
+class LinkedInCredentialsError(LinkedInError):
```

Comment:
I'm following the trend of other custom errors in the client. There is no real reason, I agree. But it would make it inconsistent not inheriting from LinkedinError.